src/main/java/fi/okm/jod/yksilo/controller/ehdotus/MahdollisuudetController.java

```diff
@@ -93,17 +87,7 @@ public List<EhdotusDto> createEhdotus(@RequestBody @Valid LuoEhdotusDto ehdotus)
 
     log.info("Creating the suggestions");
 
-    var osaamiset =
-        ehdotus.osaamiset() == null
-            ? List.<OsaaminenDto>of()
-            : osaaminenService.findBy(ehdotus.osaamiset);
-
-    var kiinnostukset =
-        ehdotus.kiinnostukset() == null
-            ? List.<OsaaminenDto>of()
-            : osaaminenService.findBy(ehdotus.kiinnostukset);
-
-    if (osaamiset.isEmpty() && kiinnostukset.isEmpty()) {
+    if (ehdotus.osaamiset.isEmpty() && ehdotus.kiinnostukset.isEmpty()) {
```

Comment:
I think that we should validate the input and not just blindly pass the URIs to the inference service (URI can be almost anything, and this is a public API)